brayns/Brayns.cpp

```diff
@@ -509,16 +507,32 @@ struct Brayns::Impl
         {
             size_t nbMaterials;
             NESTLoader loader(geometryParameters);
+
             loader.importCircuit(circuit, scene, nbMaterials);
```

Comment:
The circuit should only be loaded if Brayns can attach to the cache file. Otherwise, it's a waste of time. Also, the loading should fail is no cache file is specified.